modules/dxp/apps/portal-reports-engine-console/portal-reports-engine-console-web/src/main/java/com/liferay/portal/reports/engine/console/web/internal/admin/lar/DefinitionStagedModelDataHandler.java

```diff
@@ -124,9 +126,10 @@ protected void doExportStagedModel(
 
 			attachmentElement.addAttribute("full-file-name", fullFileName);
 
-			byte[] bytes = DLStoreUtil.getFileAsBytes(
-				definition.getCompanyId(), CompanyConstants.SYSTEM,
-				fullFileName);
+			byte[] bytes = StreamUtil.toByteArray(
+				_store.getFileAsStream(
+					definition.getCompanyId(), CompanyConstants.SYSTEM,
+					fullFileName, StringPool.BLANK));
 
 			portletDataContext.addZipEntry(binPath, bytes);
```

Comment:
Also here, no need to convert to `bytes[]` because `addZipEntry` supports `InputStream`.